aspnetcore/tutorials/first-mvc-app/adding-model.md

```diff
@@ -99,181 +106,128 @@ The preceding command adds the [aspnet-codegenerator scaffolding tool](xref:fund
 
 ---
 
-<a name="dc"></a>
-
-## Create a database context class
-
-A database context class is needed to coordinate EF Core functionality (Create, Read, Update, Delete) for the `Movie` model. The database context is derived from [Microsoft.EntityFrameworkCore.DbContext](/dotnet/api/microsoft.entityframeworkcore.dbcontext) and specifies the entities to include in the data model.
-
-Create a *Data* folder.
-
-Add a *Data/MvcMovieContext.cs* file with the following code: 
-
-[!code-csharp[](~/tutorials/first-mvc-app/start-mvc/sample/MvcMovie3/zDocOnly/MvcMovieContext.cs?name=snippet)]
-
-The preceding code creates a [DbSet\<Movie>](/dotnet/api/microsoft.entityframeworkcore.dbset-1) property for the entity set. In Entity Framework terminology, an entity set typically corresponds to a database table. An entity corresponds to a row in the table.
-
-<a name="reg"></a>
-
-## Register the database context
```

Comment:
replace below with examine

Reply:
@Rick-Anderson, what are your referring to when you say "below".  Do you mean:
1.  Replace the next ## section _below_ this with "Examine"?  So "## Add a database connection string." would become "## Examine the database connection string".
2.  Replace the section _above_ "## Register the database context" so it is "Examine the database context".

I ask since it seemed like you might have meant "above" instead since this comment is right next to that, rather close to the next section below it.

Reply:
Examine the db con str